vllm/model_executor/models/aria.py

```diff
@@ -363,27 +363,9 @@ class AriaMoELMModel(LlamaModel):
     """
 
     def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
-        super().__init__(vllm_config=vllm_config, prefix=prefix)
-
-        config = vllm_config.model_config.hf_config
-        cache_config = vllm_config.cache_config
-        quant_config = vllm_config.quant_config
-
-        # FIXME: this is a hack to disable the compilation of the model
-        self.do_not_compile = True
```

Comment:
@xffxff why do you want to disable it?

Reply:
I tested it, and it can use `torch.compile` .

Reply:
I encountered an error at the start of my PR: 'AriaMoELMModel' object has no attribute 'do_not_compile'. I didn’t investigate it further, as it might not be necessary right now. It’s great to see that we can now use `torch.compile`!